python/TestHarness/tests/test_resultsstore_storedresults.py

```diff
@@ -138,56 +152,52 @@ def convertTestHarnessResults(test_harness_results: dict, remove_header: list =
         # Setup main results entry
         results = test_harness_results.copy()
         # Dummy ID, which is the same set as 'result_id' in the tests
-        results['_id'] = RESULT_ID
+        results["_id"] = RESULT_ID
 
         # Setup the tests entry, if not instructed not to do so
-        results['tests'] = {}
+        results["tests"] = {}
         if not no_tests:
             for name, values in tests.items():
-                if name.folder not in results['tests']:
-                    results['tests'][name.folder] = {'tests': {}}
-                set_value = values.copy() if test_in_results else values['_id']
+                if name.folder not in results["tests"]:
+                    results["tests"][name.folder] = {"tests": {}}
+                set_value = values.copy() if test_in_results else values["_id"]
                 results_set_test_value(results, name, set_value)
 
         # Add in header, removing keys to be deleted
         header = deepcopy(STORE_HEADER)
-        header['civet_version'] = civet_version
+        header["civet_version"] = civet_version
         for k in remove_header:
             del header[k]
         results.update(header)
 
         return TestResultsStoredResults.ConvertedTestHarnessResults(
-            result_data=results,
-            test_data=tests
+            result_data=results, test_data=tests
         )
 
     @dataclass
     class CapturedResult(ConvertedTestHarnessResults):
-        """
-        Helper data class for captureResult()
-        """
+        """Helper data class for captureResult()."""
+
         # The test harness from the run
         harness: TestHarness
 
     def captureResult(self, **kwargs) -> CapturedResult:
-        """
-        Build TestHarness JSON output and produce data that can be used
-        to produce a StoredResult and StoredTestResult(s).
-        """
-        result = self.runTestsCached('-i', 'validation', '--capture-perf-graph', exit_code=132)
+        """Build TestHarness JSON output and produce data used for testing."""
+        result = self.runTestsCached(
+            "-i", "validation", "--capture-perf-graph", exit_code=132
+        )
 
         converted = self.convertTestHarnessResults(result.results, **kwargs)
         return self.CapturedResult(
             result_data=converted.result_data,
             test_data=converted.test_data,
-            harness=result.harness
+            harness=result.harness,
```

Comment:
Did the formatter put the trailing comma here? Same question for other instances.

Reply:
Yep. This is intentional.

If you add another argument, it'll put it on a new line and not add an extra comma so the diff just shows the new line. The formatting is targeted at limiting diffs when adding capability